__tests__/identity.js

```diff
@@ -12,6 +12,17 @@ import { URL } from 'url';
 import { URL as u } from 'whatwg-url';
 import version from '../src/version.js';
 
+import { TextEncoder, TextDecoder } from 'util';
+Object.assign(global, {TextEncoder, TextDecoder});
```

Comment:
I know this is not ideal, but I'm also working on bumping the deps and refactoring stuff like this for a separate task. I didn't want to mix this feat with the refactor, so I left it as is.